### PR TITLE
Make lxd containers ephemeral

### PR DESCRIPTION
**Pull request by evandandrea**
This fixes https://bugs.launchpad.net/snapcraft/+bug/1577548

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by elopio** [comment]
OK to test

**Comment by elopio** [comment]
Thanks @evandandrea. 

This should have an integration test, but we have an issue pending in there, so I reported https://bugs.launchpad.net/snapcraft/+bug/1593529 for later.

File: snapcraft/internal/lxd.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def _create_container(self):
             remote_tmp = petname.Generate(2, '-')
             check_call(['lxc', 'remote', 'add', remote_tmp, self._server])
             check_call([
-                'lxc', 'launch',
+                'lxc', 'launch', '-e',
```
Comment (sergiusens) [inline snapcraft/internal/lxd.py]: when did this ever get dropped? :-(

**Comment by sergiusens** [comment]
Thanks, and terribly sorry for the slow turn around on this one

**Pull request merged**